metriken-exposition/src/parquet.rs

```diff
@@ -133,56 +141,79 @@ impl ParquetSchema {
             )])),
         );
 
+        let mut counters = BTreeMap::new();
```

Comment:
I wouldn't have expected the type-inference to be able to get the type without an annotation, but that is pretty neat.